cilium-cli.yaml

```diff
@@ -0,0 +1,51 @@
+package:
+  name: cilium-cli
+  version: 0.13.0
+  epoch: 0
+  description: CLI to install, manage & troubleshoot Kubernetes clusters running Cilium
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+      paths:
+        - "*"
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - go
+      - gcc
+      - glibc-dev
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/cilium/cilium-cli
+      tag: v${{package.version}}
+      destination: cilium-cli
+      expected-commit: ac2eca8005b7bdd677272845b5d0a6261e0e8a37
+  - assertions:
+      required-steps: 1
+    pipeline:
+      - if: ${{build.arch}} == 'x86_64'

```

Comment:
Can we simplify this so it doesn't require conditionals, similar to https://github.com/wolfi-dev/os/pull/735 ? That might even help us with issues we had with `make cilium` 🤞

Reply:
Of course, cool idea 👌

Reply:
there is no target specific to the os/arch: _https://github.com/cilium/cilium-cli/blob/master/Makefile_

So, we cannot make it at the time of doing this.